packages/components/pie-input/test/component/pie-input.spec.ts

```diff
@@ -547,6 +553,53 @@ test.describe('PieInput - Component tests', () => {
                 });
             });
         });
+
+        test.describe('assistiveText', () => {
+            test('should not render the assistive text component if the prop is not provided', async ({ mount, page }) => {
+                // Arrange
+                await mount(PieInput, {});
+
+                // Act
+                const assistiveText = page.locator(assistiveTextSelector);
+
+                // Assert
+                expect((assistiveText)).not.toBeVisible();
+            });
+
+            test('should render the assistive text component if the prop is provided', async ({ mount, page }) => {
+                // Arrange
+                await mount(PieInput, {
+                    props: {
+                        assistiveText: 'Default text',
+                    } as InputProps,
+                });
+
+                // Act
+                const assistiveText = page.locator(assistiveTextSelector);
+
+                // Assert
+                expect(assistiveText).toBeVisible();
+            });
+
+            statusTypes.forEach((status) => {
```

Comment:
Please can you add a test that checks that when no status is provided, the assistive text is the default variant if it is provided?

Reply:
added